app/models/company.rb

```diff
@@ -1,2 +1,7 @@
 class Company < ActiveRecord::Base
+  validates :name,            presence: true
+  validates :description,     presence: true
+  validates :sub_description, presence: true
```

Comment:
@godfreym So for 3sixD, show me name, description, sub_description, address. I'm not seeing the distinction between the 4